packages/colorpicker/lib/src/components/color_comparison.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+class ColorComparison extends StatelessWidget {
+  const ColorComparison({
+    super.key,
+    required this.currentColor,
+    required this.newColor,
+  });
+
+  final Color currentColor;
+  final Color newColor;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 130.0,
+      height: 70.0,
+      child: Row(
+        children: [
+          Expanded(
+            child: ColorDescription(
+              color: currentColor,
+              desc: 'current',
+            ),
+          ),
+          Expanded(
+            child: ColorDescription(
+              color: newColor,
+              desc: 'new',
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class ColorDescription extends StatelessWidget {
```

Comment:
This widget does not show transparency like how slider displays. Changing the transparency in the slider make it appear brighter color in this widget instead of showing checkerboard image.